test/Tizen.NUI.Tests/Tizen.NUI.Devel.Tests/testcase/public/BaseComponents/TSTextEditor.cs

```diff
@@ -152,5 +152,35 @@ public void TextEditorEnableFontSizeScale()
             testingTarget.Dispose();
             tlog.Debug(tag, $"TextEditorEnableFontSizeScale END (OK)");
         }
-    }   
+
+        [Test]
+        [Category("P1")]
+        [Description("TextEditor Strikethrough.")]
+        [Property("SPEC", "Tizen.NUI.TextEditor.Strikethrough A")]
+        [Property("SPEC_URL", "-")]
+        [Property("CRITERIA", "CONSTR")]
+        [Property("AUTHOR", "s.sabah@samsung.com")]
+        public void TextEditorStrikethrough()
+        {
+            tlog.Debug(tag, $"TextEditorStrikethrough START");
+
+            var testingTarget = new TextEditor(true);
+            Assert.IsNotNull(testingTarget, "Can't create success object TextEditor");
+            Assert.IsInstanceOf<TextEditor>(testingTarget, "Should be an instance of TextEditor type.");
+
+            using (PropertyMap map = new PropertyMap())
```

Comment:
Hello @shrouqsabah 

Thanks for adding the test :)
But this test doesn't seem appropriate.

As you know, there is no Strikethrough Property.

I think that the `map <-> struct` convert test is sufficient as the TSTextMapHelper test,
but if you add a test here, you need to verify the Set/Get method.
Maybe you could do a test like this:
```
var underline = new Tizen.NUI.Text.Strikethrough();
underline.Enable = true;
underline.Color = new Color("#3498DB");
underline.Height = 5.0f;

testingTarget.SetStrikethrough(underline);
var getUnderline = underlineLabel.GetStrikethrough();

Assert.AreEqual(underline.Enable, getUnderline.Enable, "Should be equal!");
Assert.AreEqual(underline.Height, getUnderline.Height, "Should be equal!");
Assert.AreEqual(true, CheckColor(underline.Color, getUnderline.Color), "Should be true!");
```

Reply:
Yes, you are right. 
Could you check the new one. please.
I added set/get in one testcase but SPEC for Get ("SPEC", "Tizen.NUI.TextField.GetStrikethrough M").
Is that OK ?

Reply:
Looks good, thanks :)